raiden/accounts.py

```diff
@@ -10,10 +10,12 @@
 def find_datadir():
     home = os.path.expanduser("~")
     datadir = None
-    if sys.platform.startswith('linux'):
+    if sys.platform.startswith('linux') or sys.platform.startswith('FreeBSD'):
```

Comment:
Just to add more confusion IMO the [`platform`](https://docs.python.org/2/library/platform.html) module is the recommended way of getting platform information